sonar-java-plugin/src/test/java/org/sonar/plugins/java/JavaSquidSensorTest.java

```diff
@@ -73,11 +74,13 @@ public void setUp() {
   }
 
   @Test
+  @Ignore
   public void test_issues_creation_on_main_file() throws IOException {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Fix or remove this skipped unit test [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1607)